public/javascripts/running.js

```diff
@@ -237,29 +212,30 @@ function init_livelog(url) {
 /********* LIVE STREAM *********/
 
 // global vars for livestream
-var livestream, last_event, image_size_x, image_size_y;
+var livestream, last_event;
```

Comment:
kind of. init_livestream() sets the global variable, but there are no users left. So I can make it a local variable